pkgs/data/themes/arc/default.nix

```
@@ -11,19 +12,23 @@
 , cinnamon
 }:
 
-stdenv.mkDerivation rec {
+let
+  transparency = true;
+  gnomeShellResource = false;
+in stdenv.mkDerivation rec {
   pname = "arc-theme";
-  version = "20210127";
+  version = "20210412";
 
   src = fetchFromGitHub {
     owner = "jnsh";
     repo = pname;
     rev = version;
-    sha256 = "sha256-P7YZTD5bAWNWepL7qsZZAMf8ujzNbHOj/SLx8Fw3bi4=";
+    sha256 = "sha256-BNJirtBtdWsIzQfsJsZzg1zFbJEzZPq1j2qZ+1QjRH8=";
```

```suggestion
    hash = "sha256:BNJirtBtdWsIzQfsJsZzg1zFbJEzZPq1j2qZ+1QjRH8=";
```

I think this is the latest style.

---

Is there any doc around this?

---

The colon is the old style. Dash is the new style, using the https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity format.

---

And I would not use the `hash` attribute yet, since [`fetchgit` does not support it](https://github.com/NixOS/nixpkgs/pull/79987).